### DISCUSSION TITLE
Support Jira Service Management / Service Desk

### DISCUSSION THREAD

**Discussion by avthart** [ankitpokhrel/jira-cli#519] [Ideas]
Would be nice if the Jira CLI can support creating changes, incidents, etc. 

Currently not working because the Epic field is set (also when it's null).

```
❯ jira issue create --no-input -t'[System] Change' -s "test change"

Error:
  - customfield_10014: Field 'customfield_10014' cannot be set. It is not on the appropriate screen, or unknown.

jira: Received unexpected response '400 Bad Request'.
Please check the parameters you supplied and try again.
```

The field `customfield_10014` is related to the epic field.

**Comment by ankitpokhrel** [comment]
Hi @avthart, wondering how is this related to Jira service desk? 

The error you see is from the Jira API and its complaining that the field `customfield_10014` is not available in the screen. Is this a required field when creating an issue? Could you provide some more details about your workflow please.
